File: Independent Unity Project/Scripts for Go Nuts/Rot.cs
using UnityEngine;
using System.Collections;

public class Rot : MonoBehaviour
{


	// Update is called once per frame
	void Update ()
	{
	
		//CHANGE THIS SO THAT THE NUT SHOT WILL BE OUT OF THIS CONTROL

		Vector3 mousePos = Input.mousePosition;
		mousePos.z = transform.position.z - Camera.main.transform.position.z;
		Vector3 poss = Camera.main.WorldToScreenPoint (transform.position);
		Vector3 dir = mousePos - poss;
		float angle = Mathf.Atan2 (dir.y, dir.x) * Mathf.Rad2Deg;
		//transform.rotation = Quaternion.AngleAxis (angle, Vector3.forward);
		transform.rotation = Quaternion.Euler (new Vector3 (0, 0, angle));


	}
}

File: Independent Unity Project/Go Nuts(Proto)/Assets/Scripts/rot.js
#pragma strict

var mouse_pos:Vector3;
var target:Transform;
var object_pos:Vector3;
static var mouseAngle:float;


function Start () {

}

function Update () {
	        //follow mouse
        mouse_pos = Input.mousePosition;
        mouse_pos.z = 0; //The distance between the camera and object
        object_pos = Camera.main.WorldToScreenPoint(target.position);
        mouse_pos.x = mouse_pos.x - object_pos.x;
        mouse_pos.y = mouse_pos.y - object_pos.y;
        mouseAngle = Mathf.Atan2(mouse_pos.y, mouse_pos.x) * Mathf.Rad2Deg;
        transform.rotation = Quaternion.Euler(Vector3(0, 0, mouseAngle));
 
 		/*
        transform.position.x=left_hand.position.x;
        transform.position.y=left_hand.position.y;
        transform.position.z=left_hand.position.z;
		*/
}